idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -962,17 +962,17 @@ def entropy_correlation(b):
                 b.pressure,
                 to_units=pyunits.Pa)
             r_gas = pyunits.convert(
-                b._params.gas_const,
+                Constants.gas_constant,
                 to_units=pyunits.J/pyunits.mol/pyunits.K)
             return (
                 (self.entr_mol + r_gas * log(p/1e5)) * b.flow_mol ==
                 sum(b.flow_mol*b.mole_frac_comp[j] * (
-                    self.params.cp_mol_ig_comp_coeff_A[j]*log(t) +
-                    self.params.cp_mol_ig_comp_coeff_B[j]*t +
-                    self.params.cp_mol_ig_comp_coeff_C[j]*t**2 / 2 +
-                    self.params.cp_mol_ig_comp_coeff_D[j]*t**3 / 3 -
-                    self.params.cp_mol_ig_comp_coeff_E[j]/t**2 / 2 +
-                    self.params.cp_mol_ig_comp_coeff_G[j] +
+                    self.params.cp_param_1[j]*log(t) +
```

Comment:
This looks like there would be a unit consistency issue. From what I can tell, `t` has units of `kK`, which means that the `log()` term here will have a unit consistency failure.

1. In these cases, you just need to do `log(t/pyunits.kK)`.
2. The fact that we don't see a test failure implies that we do not have a test for unit consistency. Is this an oversight, or are we not quite ready for unit consistency tests overall?

Reply:
@bpaul4 similar to the challenges we had with the scaling tests, it is possible that the property constraints have not been constructed therefore the units consistency check might not be testing the constraints. If that's the case, then the property constraints will have to be constructed like was done in the scaling tests before calling the assert_units_consistent method.